app/scripts/lib/multichain-api/scope/validation.ts

```diff
@@ -67,22 +67,6 @@ export const isValidScope = (
     return false;
   }
 
-  // Note we are not validating the chainId here, only the namespace
-  // const areAccountsValid = (accounts || []).every((account) => {
-  //   try {
-  //     return parseCaipAccountId(account).chain.namespace === namespace;
-  //   } catch (e) {
-  //     // parsing caipAccountId failed
-  //     console.log(e);
-  //     return false;
-  //   }
-  // });
-
-  // if (!areAccountsValid) {
-  //   return false;
-  // }
-  // not validating rpcDocuments or rpcEndpoints currently
-
```

Comment:
I think we should still validate that the CAIP-10 account address is formed correctly and matches the namespace of the scopeString

At least I believe the following should cause an error
```
{
  "eip155:1": {
    ...,
    accounts:['bip122:000000000019d6689c085ae165831e93:128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6
']
 }
}
```